src/AspNet.Security.OAuth.Moodle/MoodleAuthenticationDefaults.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+
+namespace AspNet.Security.OAuth.Moodle
+{
+    /// <summary>
+    /// Default values used by the Moodle authentication middleware.
+    /// </summary>
+    public class MoodleAuthenticationDefaults
+    {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationScheme.Name"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "Moodle";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationScheme.DisplayName"/>.
+        /// </summary>
+        public static readonly string DisplayName = "Moodle";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationSchemeOptions.ClaimsIssuer"/>.
+        /// </summary>
+        public const string Issuer = "Moodle";
+
+        /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
+        /// </summary>
+        public const string CallbackPath = "/signin-moodle";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
+        /// </summary>
+        public static readonly string AuthorizationEndpoint = "http://moodledomain.com/local/oauth/login.php";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.TokenEndpoint"/>.
+        /// </summary>
+        public static readonly string TokenEndpoint = "http://moodledomain.com/local/oauth/token.php";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.UserInformationEndpoint"/>.
+        /// </summary>
+        public static readonly string UserInformationEndpoint = "http://moodledomain.com/local/oauth/user_info.php";
```

Comment:
There's no point in including these constants if they point to a domain that doesn't exist.

See the Okta provider for an example of a provider that works against user-provided domains:

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/4b83b0a361197290d1faa8f14fc0d59bcdeeaad9/src/AspNet.Security.OAuth.Okta/OktaAuthenticationDefaults.cs#L32-L50